test-programs/e2e-test/tests/test.rs

```diff
@@ -1,28 +1,32 @@
 #![cfg(feature = "test-sbf")]
 
+use anchor_lang::solana_program::native_token::LAMPORTS_PER_SOL;
 use light_test_utils::e2e_test_env::{E2ETestEnv, GeneralActionConfig, KeypairActionConfig};
+use light_test_utils::rpc::rpc_connection::RpcConnection;
 use light_test_utils::rpc::{ProgramTestRpcConnection, SolanaRpcConnection};
 use light_test_utils::test_env::{get_test_env_accounts, setup_test_programs_with_accounts};
+use solana_sdk::signature::Signer;
 use std::process::Command;
 
 async fn spawn_test_validator() {
-    println!("Starting validator...");
-    let path = "../../cli/test_bin/run test-validator --skip-indexer --skip-prover";
+    let path = "../../cli/test_bin/run test-validator --skip-indexer --skip-prover --skip-forester";
     Command::new("sh")
         .arg("-c")
         .arg(path)
         .spawn()
         .expect("Failed to start server process");
     tokio::time::sleep(tokio::time::Duration::from_secs(10)).await;
-    println!("Validator started successfully");
 }
 
 #[tokio::test(flavor = "multi_thread", worker_threads = 4)]
 async fn test_10_validator_all() {
     spawn_test_validator().await;
     let env_accounts = get_test_env_accounts();
     let rpc = SolanaRpcConnection::new(None);
-
+    rpc.client
+        .request_airdrop(&rpc.get_payer().pubkey(), LAMPORTS_PER_SOL * 1000)
+        .unwrap();
+    tokio::time::sleep(std::time::Duration::from_secs(16)).await;
```

Comment:
Iirc, @ananas-block discovered that we can use a random keypair here for the payer as a hotfix. The underlying issue is that the same hardcoded keypair gets used elsewhere in a completely unrelated test at the same time, and therefore, the airdrop here updates its balance during that other test run, causing that other test to fail.